i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/create-your-connector/connector-implementation-guide.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: 搭建连接器指南
+sidebar_position: 2
+---
+
+import ContentsInConstant from './fragments/_constant_contents.md';
+import ContentsInIndex from './fragments/_index_contents.md';
+import ContentsInTypes from './fragments/_types_contents.md';
+import ContentsInUtils from './fragments/_utils_contents.md';
+
+# 搭建连接器指南
+
+对连接器的文件结构有了一个初步概念之后，让我们讨论一下开发、实现一个连接器的思路。
+
+## 在连接器对接的服务商网站注册一个账号
+
+仔细浏览服务商的网站和相应的管理页面，总是我们开始实现一个连接器的第一步。在这里你可以找到关于连接器服务提供商的一些基本信息。
+
+## 查找讲解连接器要对接的服务的功能和工作流程的文档
+
+由于不同类型的连接器对应着不同的使用场景，我们在搜索相关的文档的时候也需要注意用不同的关键词。
+
+_社交连接器_ 是用于引导终端用户通过 OAuth 2.0 协议授权登录的；而 _短信连接器_ 和 _邮件连接器_ 适用于调用发送短信或邮件服务 API 的。
```

Comment:
i think not all connectors are using oauth 2

Reply:
updated